application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java

```diff
@@ -188,4 +198,12 @@ private static ForumTag getDefaultTagOr(List<ForumTag> tagsFoundOnForum,
             Supplier<ForumTag> defaultTag) {
         return tagsFoundOnForum.isEmpty() ? defaultTag.get() : tagsFoundOnForum.get(0);
     }
+
+    private MessageEmbed makeEmbedForPost(User originalUser, String modifiedMessage) {
+        return new EmbedBuilder().setAuthor(originalUser.getName())
+            .setThumbnail(originalUser.getAvatarUrl())
+            .setDescription(modifiedMessage)
+            .setColor(1752220)
```

Comment:
unreadable color. use `Color.decodeHex(...)` or whatever its called. that also has IDE support. and put it into a constant